_includes/v19.1/app/insecure/txn-sample.clj

```diff
@@ -0,0 +1,43 @@
+(ns test.test
+  (:require [clojure.java.jdbc :as j]
+            [test.util :as util]))
+
+;; Define the connection parameters to the cluster.
+(def db-spec {:subprotocol "postgresql"
+              :subname "//localhost:26257/bank"
+              :user "maxroach"
+              :password ""})
```

Comment:
Is this `:password` key needed for the insecure examples?  If it's not I think we should drop it here also, since we're still not doing pw-based auth (in fact I don't think CRDB really does pw-based auth, IIRC)

Reply:
I ran into some "password required" errors, but I just ran through both examples again with the password parameter and everything worked. Must have been for the secure example when I was using the wrong type of key. I'll remove that parameter from both!